github.go

```diff
@@ -104,7 +104,7 @@ func FetchAllIssues() *QueryPages {
 		query := Query{}
 		err := client.Query(context.Background(), &query, variables)
 		if err != nil {
-			log.Fatal(err)
+			log.Error(err)
```

Comment:
Shouldn't we just fail had here, since there is obviously something wrong with fetching the data?

Reply:
Well, it depends. If its something permanent (e.g invalid credentials), yes. But there are other cases (e.g. temporary connection failures) that we might want to tolerate. A call to log.Fatal will always kill the program no matter what.

Reply:
Well, I guess this should be handled here! :laughing: 
https://github.com/brejoc/filtra/issues/7